From 508b5aed8f2bd90ead0e8d08ac9804efa502f516 Mon Sep 17 00:00:00 2001
From: Calum Lind <calumlind+deluge@gmail.com>
Date: Sun, 25 Nov 2012 23:30:13 +0000
Subject: [PATCH] Fix #2201 : auth fails if auth file has extra newlines

---
 deluge/core/authmanager.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/deluge/core/authmanager.py b/deluge/core/authmanager.py
index ef6fbc4..235e254 100644
--- a/deluge/core/authmanager.py
+++ b/deluge/core/authmanager.py
@@ -121,10 +121,10 @@ def __load_auth_file(self):
             f = open(auth_file, "r").readlines()
 
         for line in f:
-            if line.startswith("#"):
-                # This is a comment line
-                continue
             line = line.strip()
+            if line.startswith("#") or not line:
+                # This line is a comment or empty
+                continue
             try:
                 lsplit = line.split(":")
             except Exception, e:
-- 
1.8.0.2

